src/components/InfoBanner/InfoBanner.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Layout from './Layout';
+import Highlight from './Highlight';
+
+function InfoBanner() {
+  return (
+    <Layout>
+      You have received&nbsp;<Highlight>free shipping</Highlight>
```

Comment:
I would make it a bit more flexible by accepting children. So in nutshell it would be banner with colored background, which styles passed as children text.
Use case:
<InfoBanner>You have received <strong>free shipping</strong>!</InfoBanner>